doc/user/static.rst

```diff
@@ -176,3 +176,52 @@ multiple segments instructions.
   router# show ipv6 route
   [..]
   S>* 2005::/64 [1/0] is directly connected, ens3, seg6 2001:db8:aaaa::7,2002::4,2002::3,2002::2, weight 1, 00:00:06
+
+SRv6 Static SIDs Commands
+=========================
+
+.. clicmd:: segment-routing
+
+   Move from configure mode to segment-routing node.
+
+.. clicmd:: srv6
+
+   Move from segment-routing node to srv6 node.
+
+.. clicmd:: static-sids
+
+   Move from srv6 node to static-sids node. In this static-sids node, user can
+   configure static SRv6 SIDs.
+
+.. clicmd:: sid X:X::X:X/M locator NAME behavior <uN|uDT4|uDT6|uDT46> [vrf VRF]
+
+   Specify the locator sid manually. Configuring a local sid in a purely static mode
+   by specifying the sid value would generate a unique SID.
+   This feature will support the configuration of static SRv6 decapsulation on the system.
+
+   It supports four parameter options, corresponding to the following functions:
+   uN, uDT4, uDT6, uDT46
+
+   When configuring the local sid, if the action is set to 'uN', no vrf should be set.
+   While for any other action, it is necessary to specify a specific vrf.
+
+::
+
+   router# configure terminal
+   router(config)# segment-routing
+   router(config-sr)# srv6
+   router(config-srv6)# static-sids
+   router(config-srv6-sids)# sid fcbb:bbbb:1:fe01::/64 locator LOC1 behavior uDT6 vrf Vrf1
+   router(config-srv6-sids)# sid fcbb:bbbb:1:fe02::/64 locator LOC1 behavior uDT4 vrf Vrf1
+   router(config-srv6-sids)# sid fcbb:bbbb:1:fe03::/64 locator LOC1 behavior uDT46 vrf Vrf2
```

Comment:
I dont understand the API. Sorry:
the SID help is : 
```
ubuntu2204hwe(config-srv6-sids)# sid 
  X:X::X:X/M  Specify SRv6 SID
```
From my perspective, a SID is a 128 bit value.
So why having mask ?

With the current knowledge I have, I would require that the sid parameter be an ipv6 address and not a prefix

Reply:
/64 for usid looks ok for me. 
Example: https://github.com/FRRouting/frr/pull/13724/commits/6d22a6e5db1887c0ef91a4da3da479f049421ddb#diff-7878a0c33223f5a550e973671d444b50c42c33e2b7aaa9144d53ae12411b610dR226